libr/bin/format/mach0/mach0.c

```diff
@@ -1744,6 +1744,162 @@ static int inSymtab(SdbHt *hash, struct symbol_t *symbols, const char *name, ut6
 	return false;
 }
 
+void MACH0_(pull_symbols)(struct MACH0_(obj_t)* mo, RBinSymbolCallback cb, void *user) {
+	const char *symstr;
+	int j, s, stridx, symbols_size, symbols_count;
+	ut32 to, from, i;
+	//ut64 text_base = get_text_base (bin);
+
+	if (!mo || !mo->symtab || !mo->symstr) {
+		return;
+	}
+	/* parse symbol table */
+	/* parse dynamic symbol table */
+	symbols_count = (mo->dysymtab.nextdefsym + \
+			mo->dysymtab.nlocalsym + \
+			mo->dysymtab.nundefsym );
+	symbols_count += mo->nsymtab;
+	//symbols_count = mo->nsymtab;
+	symbols_size = (symbols_count + 1) * 2 * sizeof (struct symbol_t);
+
+	if (symbols_size < 1) {
```

Comment:
AFAIK this conditions should never happen. `symbols_count` is the sum of uint32_t numbers so it can never be `-1`, which is the only value to make `symbols_size < 1`. Unless you are checking for overflows...

Reply:
it's used to catch some fuzzed bins, checking this may require some careful eye on this. i will not touch this in this pr, but feel free o discuss that in a separate issue